lib/average_daily_balance/main.py

```diff
@@ -0,0 +1,117 @@
+import os
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd
+from pngme.api import Client
+
+
+def get_avg_daily_balance(
```

Comment:
Is this better to be named as `get_avg_end_of_day_daily_balance`, as it computes the `end of day` balance?
May also need to update the docstring then.

Reply:
name too long and confusing

Reply:
Looks like the definition of [average daily balance](https://www.investopedia.com/terms/a/averagedailybalance.asp) implies that it is the average of balances at the end of each day.

Is it safe to assume domain experts would be familiar with the end-of-day assumption? If so, probably fine to leave as-is.

Reply:
end-of-day is a domain assumption, so fine to leave as is